interpreter/cling/test/Prompt/ValuePrinter/Collections.C

```diff
@@ -26,7 +26,7 @@ for (int i = 0; i < 5; ++i) {
 }
 Bvv.push_back(std::vector<bool>());
 Bvv
-// CHECK-NEXT: (std::vector<std::vector<bool> > &) { { false }, { false, true }, { false, true, true }, { false, true, true, true }, { false, true, true, true, false }, {} }
+// CHECK-NEXT: (std::vector<std::vector<bool>> &) { { false }, { false, true }, { false, true, true }, { false, true, true, true }, { false, true, true, true, false }, {} }
```

Comment:
I used the adjustment in the template printing policy. See 080d3e4f21b - maybe we should do the same and hope to fix a bunch of potential reference files in roottest or experiments?

Reply:
Ok, we can also do this... However, I'd agree with https://github.com/llvm/llvm-project/commit/159a9f7e76307734bcdcae3357640e42e0733194: It's not 1998 anymore.

Reply:
Sure ;), if we want to move to the new century we should edit a lot of text files. I am not opposing this, I am just saying that can spawn between multiple repositories including experiments…

Reply:
Addressed via the printing policy as discussed this morning